TaskForge.NET/TaskForge.WebUI/wwwroot/js/project-dashboard/task-edit.js

```
@@ -158,11 +166,16 @@ function RemoveAttachment(attachmentId) {
+        success: function (response) {
             // remove the list item from UI without full reload
-            $(`#task-attachment-list button[onclick="RemoveAttachment(${attachmentId})"]`)
-                .closest('li')
-                .remove();
+            if (response.success) {
+                $(`#task-attachment-list button[onclick="RemoveAttachment(${attachmentId})"]`)
+                    .closest('li')
+                    .remove();
+            }
+            else {
+                alert("Failed to delete attachment: " + response.message);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)